services/painting-process-equipment-simulator-service/app/services/model_client.py

```
@@ -0,0 +1,100 @@
+        service_url = settings.model_services.get(service_name)
+
+        if not service_url:
+            return False
+
+        health_url = f"{service_url}/health"
+
+        try:
+            async with httpx.AsyncClient(timeout=httpx.Timeout(5.0)) as client:
+                response = await client.get(health_url)
+                return response.status_code == 200
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)